k8s.go

```diff
@@ -25,20 +25,22 @@ type Cert string
 
 // These constants used as Cert parsing a secret received from the API.
 const (
-	APICert            Cert = "api"
-	CalicoCert         Cert = "calico"
-	EtcdCert           Cert = "etcd"
-	FlanneldCert       Cert = "flanneld"
-	NodeOperatorCert   Cert = "node-operator"
-	PrometheusCert     Cert = "prometheus"
-	ServiceAccountCert Cert = "service-account"
-	WorkerCert         Cert = "worker"
+	APICert             Cert = "api"
+	CalicoCert          Cert = "calico"
+	ClusterOperatorCert Cert = "cluster-operator"
```

Comment:
Shouldn't this be `cluster-operator-api` in case we add more?

Reply:
Yes that is better. Done.